app/controllers/recipe_foods_controller.rb

```diff
@@ -49,10 +49,11 @@ def update
 
```

Comment:
- [ ] @team kindly ensure that the owners of the recipe(this can also be applied to food) are the only ones who can delete the recipes they created, this a project requirement:
![image](https://user-images.githubusercontent.com/61408860/202848392-bf8348d4-f7be-438d-b6df-81c40faa24f4.png)

Reply:
done